Source/SourceKittenFramework/Request.swift

```diff
@@ -340,3 +340,70 @@ extension Request: CustomStringConvertible {
     /// A textual representation of `Request`.
     public var description: String { return String(UTF8String: sourcekitd_request_description_copy(sourcekitObject))! }
 }
+
+private func interfaceForModule(module: String) -> [String: SourceKitRepresentable] {
+    let xctestConfigurationPath = NSProcessInfo.processInfo().environment["XCTestConfigurationFilePath"]!
+    let buildPath = xctestConfigurationPath.substringToIndex(xctestConfigurationPath.rangeOfString("/Intermediates/")!.startIndex)
```

Comment:
On that condition, `XCTestConfigurationFilePath` is `/Users/norio/Library/Developer/Xcode/DerivedData/SwiftLint-eangswdzvhtbxpebxskdhgwvjger/Build/Products/Debug/SourceKittenFrameworkTests.xctest/Contents/Resources/SourceKittenFrameworkTests-BFFF5841-89B0-4790-A8D7-61359F29A156.xctestconfiguration`

On running test of sourcekitten scheme inside `SourceKitten.xcworkspace`, `XCTestConfigurationFilePath` is `/Users/norio/Library/Developer/Xcode/DerivedData/SourceKitten-drsebuetmrpoaedcrikulbrsfzey/Build/Products/Debug/SourceKittenFrameworkTests.xctest/Contents/Resources/SourceKittenFrameworkTests-E6113DED-C71F-471C-931D-47462DC42DEF.xctestconfiguration`